Makefile

```diff
@@ -0,0 +1,25 @@
+SHELL := /bin/bash -e
+
+ifneq ($(wildcard .env),)
```

Comment:
What're we injecting env vars for? Do one of the dependencies make use of something?